src/main/java/com/box/sdk/BoxAPIResponseException.java

```diff
@@ -40,8 +40,16 @@ public BoxAPIResponseException(String message, BoxAPIResponse responseObj) {
         try {
             responseJSON = JsonObject.readFrom(responseObj.bodyToString());
 
-            if (responseObj.bodyToString() != null && responseJSON.get("request_id") != null) {
-                requestId = " | " + responseJSON.get("request_id").asString();
+            if (responseObj.bodyToString() != null && responseJSON.get("request_id") != null
```

Comment:
You probably don't need to repeat the conditions from inside this branch here in the outer conditional — can you flatten this into two top-level conditions to construct `requestID`?